src/main/java/com/battleship/backend/BoardRepository.java

```diff
@@ -16,15 +17,15 @@ public BoardRepository() {
         boards = new Board[] { playerBoard, computerBoard };
     }
 
-    public Board[] getBoards() {
+    public Boardable[] getBoards() {
```

Comment:
Even though these models are pretty straightforward, let's add tests for all of these (including the BoardRepository) to get in the habit of keeping code coverage high.

Reply:
This is my fault. I had told Lane previously that it's not necessary to write tests for setters and getters.  But, as @pumpkincouture recommends, good to get into the practice of writing tests.

Reply:
Ok, will add those back now!

Reply:
And here's the scoop on getters and setters from some reading:  if they don't contain any logic then you can indirectly test them by using them when you test other methods.  If they contain logic (then they're actually not getters or setters), then a good idea to have tests for them.